partiql-planner/src/main/kotlin/org/partiql/planner/internal/fn/builtins/FnEq.kt

```diff
@@ -49,7 +47,7 @@ internal object Fn_EQ__ANY_ANY__BOOL : Fn {
     override fun invoke(args: Array<PartiQLValue>): PartiQLValue {
         val lhs = args[0]
         val rhs = args[1]
-        if (lhs.type == MISSING || rhs.type == MISSING) {
+        if (lhs.type == ANY || rhs.type == ANY) {
```

Comment:
Why ANY?

Reply:
Update, these were automatically changed by IntelliJ due to the "replaceWith" on the MISSING deprecated annotation.